app/src/main/java/org/jellyfin/mobile/settings/SettingsActivity.kt

```diff
@@ -18,6 +18,7 @@ class SettingsActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        setTitle(R.string.activity_name_settings)
```

Comment:
This shouldn't be necessary, as it's already set in the manifest.

Reply:
Unfortunately it's necessary to load a new translation or something like that.

**Without setTtile**
![image](https://user-images.githubusercontent.com/25919231/92948213-a2751600-f41e-11ea-865c-a1e50915d61c.png)


**With setTitle**
![image](https://user-images.githubusercontent.com/25919231/92948223-a6a13380-f41e-11ea-827b-29ebc5f5ffd5.png)